Assets/HoloToolkit/Utilities/Tests/Scripts/GestureResponder.cs

```diff
@@ -1,11 +1,31 @@
-﻿using UnityEngine;
-using System.Collections;
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
The CLA just establishes a _license_ for Microsoft (and others) to use/distribute/etc. your submitted work. They do not require transfer of copyright/ownership -- you keep that!

> a. Copyright License. You grant Microsoft, and those who receive the Submission directly or indirectly from Microsoft, a perpetual, worldwide, non-exclusive, royalty-free, irrevocable license in the Submission to reproduce, prepare derivative works of, publicly display, publicly perform, and distribute the Submission and such derivative works, and to sublicense any or all of the foregoing rights to third parties.

In this case, I just wasn't sure who owned the file. The owner, or Microsoft, should be the one to document the owner. But you're probably 100% right that this piece of code is owned by Microsoft.